actions/configure/parse/parse.sh

```diff
@@ -4,16 +4,21 @@ set -euo pipefail
 
 export D2L_TF_CONFIGURE_TMP_DIR=$(mktemp -d)
 
+role_suffix=${GITHUB_REPOSITORY/'/'/+}
+role_suffix=${role_suffix/#'BrightspaceHypermediaComponents'/'BHC'}
+
 echo "${CONFIG}" \
 	| yq -o=json . - \
 	| jq -cr '
 		.[]
 		| . as $account
+		| (.provider_role_arn_ro // "arn:aws:iam::\($account.account_id):role/terraform/tfp+github+'"${role_suffix}"'") as $provider_role_arn_ro
+		| (.provider_role_arn_rw // "arn:aws:iam::\($account.account_id):role/terraform/tfa+github+'"${role_suffix}"'") as $provider_role_arn_rw
 		| $account.workspaces[]
-		| .provider_role_tfvar // $account.provider_role_tfvar // "terraform_role_arn" as $tfvar
+		| (.provider_role_tfvar // $account.provider_role_tfvar // "terraform_role_arn") as $tfvar
```

Comment:
this is a bug fix. Without the parentheses, `.provider_role_tfvar` will be returned at the end (without invoking the `as $tfvar` part)